packages/react-components/src/components/MessageThread.tsx

```diff
@@ -178,18 +182,43 @@ const DefaultJumpToNewMessageButton = (props: JumpToNewMessageButtonProps): JSX.
   );
 };
 
+const generateParticipantsStr = (participants: CommunicationParticipant[]): string =>
+  participants.reduce(
+    (previous, current): string => (current.displayName ? `${previous}${current.displayName} ` : previous),
+    ''
+  );
+
 export type DefaultMessageRendererType = (props: MessageProps, ids?: { messageTimestamp?: string }) => JSX.Element;
 
 const DefaultSystemMessageRenderer: DefaultMessageRendererType = (props: MessageProps) => {
+  const { strings } = useLocale();
   if (props.message.type === 'system') {
-    const payload: SystemMessagePayload = props.message.payload;
-    return (
-      <SystemMessageComponent
-        iconName={(payload.iconName ?? '') as SystemMessageIconTypes}
-        content={payload.content ?? ''}
-        containerStyle={props?.messageContainerStyle}
-      />
-    );
+    const payload = props.message.payload;
+    if (payload.type === 'content') {
+      return (
+        <SystemMessageComponent
+          iconName={(payload.iconName ?? '') as SystemMessageIconTypes}
+          content={payload.content ?? ''}
+          containerStyle={props?.messageContainerStyle}
+        />
+      );
+    } else if (payload.type === 'participantAdded') {
```

Comment:
nit: no need for 'else if' here. 'if' can be used only as you already return a value in previous 'if'.